### ISSUE TITLE
repo segmented data storage

### ISSUE THREAD

**Issue by alexgchap** [artemkin/git-lfs-server#12]
Hi,

is there a way for the lfs server to store the hash files in a different place depending on the repo that from where I am pushing to the server?

**Comment by artemkin** [comment]
No, there is no such option. The easiest way is to run several instances of lfs server.
